apps/platform-app/src/app/[locale]/(profile)/(components)/LocaleSwitcher/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,10 @@
+import { I18nNamespaces } from '@/i18n'
+import { useLocale, useTranslations } from 'next-intl'
+import { LocaleSwitcherSelect } from './LocaleSwitcherSelect'
+
+export const LocaleSwitcher = () => {
+  const t = useTranslations(I18nNamespaces.LocaleSwitcher)
+  const locale = useLocale()
```

Comment:
If you use hooks inside a component, please make it a client component. "use client"